packages/core/contracts/oracle/implementation/test/VotingV2Test.sol

```diff
@@ -10,10 +10,11 @@ contract VotingV2ControllableTiming is VotingV2, Testable {
     constructor(
         uint256 _emissionRate,
         uint256 _spamDeletionProposalBond,
-        uint64 _unstakeCoolDown,
-        uint64 _phaseLength,
-        uint64 _minRollToNextRoundLength,
+        uint256 _unstakeCoolDown,
+        uint256 _phaseLength,
+        uint256 _minRollToNextRoundLength,
```

Comment:
I think these will need to be uint64 to match the previous packing changes. This might've been accidentally reverted.